urls.py

```diff
@@ -36,7 +40,18 @@
         'https://s3-us-west-2.amazonaws.com/qiime2-data/distro/core/qiime20174-1492814818.zip',
     'distro/core/qiime20174-1492814818.zip':
         'https://s3-us-west-2.amazonaws.com/qiime2-data/distro/core/qiime20174-1492814818.zip',
-    
+
+    # 2017.5 DISTRO
+    'distro/core/qiime2-2017.5-conda-osx-64.txt':
+        'https://raw.githubusercontent.com/qiime2/environment-files/master/2017.5/release/qiime2-2017.5-conda-osx-64.txt',
+    'distro/core/qiime2-2017.5-conda-linux-64.txt':
+        'https://raw.githubusercontent.com/qiime2/environment-files/master/2017.5/release/qiime2-2017.5-conda-linux-64.txt',
+    ### TODO: Update these URLs
```

Comment:
Actually, do we *need* these to be data links? We could just put the S3 links into distro/core/virtualbox-images.txt directly, then it wouldn't require a 2-part PR for version bumps.